pcdet/models/backbones_3d/vfe/dynamic_pillar_vfe.py

```diff
@@ -140,3 +140,101 @@ def forward(self, batch_dict, **kwargs):
         batch_dict['pillar_features'] = features
         batch_dict['voxel_coords'] = voxel_coords
         return batch_dict
+
+
+class DynamicPillarPFE(VFETemplate):
```

Comment:
What's the difference between this DynamicPillarPFE and  the original DynamicPillarVFE? 
Can we use the original DynamicPillarVFE and add some config to support the function of `DynamicPillarPFE`?

Reply:
DynamicPillarPFE  is the simplied version for dynamic pillar representation, where f_cluster is also dropped and voxel_coords is 3-dimensions (byx) for spconv 2D.
Also, renaming voxel coords as pillar coords may be better.